### PR TITLE
2nd iCBT: Royal Mausoleum and other minor additions and fixes

### PR DESCRIPTION
**Pull request by Sunaries**
2nd iCBT: Royal Mausoleum and other minor additions and fixes

Also:
- capitalized 'revelation' to 'Revelation'.
- changed tooltip for 'Divine Might' skill. Now it says: "Applies #{SkillFactor}# times". Players were confused about this, so I changed it to: "Applies for #{SkillFactor}#Skill Uses"

@imcgames Is it correct that Divine Might increases skill levels by 1 up to a certain uses? 

Example)

Player A cast Lv1 Divine Might (applies 2 times) 
Player B in party receives the buff and his skill levels are increased by 1. So, from Lv1 Bash he gets Lv2 Bash but the Lv2 disappears after he uses it 2 times, correct?

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -10534,15 +10534,15 @@ QUEST_LV_0100_20151022_010529
...
 QUEST_LV_0100_20151022_010534	
 QUEST_LV_0100_20151022_010535	
-QUEST_LV_0100_20151022_010536	
+QUEST_LV_0100_20151022_010536	$Saving the Guide Owl
 QUEST_LV_0100_20151022_010537	
 QUEST_LV_0100_20151022_010538	
 QUEST_LV_0100_20151022_010539	
 QUEST_LV_0100_20151022_010540	
-QUEST_LV_0100_20151022_010541	
+QUEST_LV_0100_20151022_010541	$Sacred Tree of the Forest (1)
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Note to self: Wrong (1) needs to be (2)

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**